enrichment_logbook_uploader/logbook_upload.py

```diff
@@ -109,6 +110,33 @@ def generate_payload(df, logbook_header_id):
         for date in skipped_dates:
             print(date)
 
+    get_logBook_response = requests.post(_ACTIVITY_ENRICHMENT_URL + "/LogBook/GetLogBook", headers=headers, data="logBookHeaderID="+logbook_header_id)
+
+    emptyCounter = 0
+    dateList = {}
+    date = 0
+    for days in json.loads(get_logBook_response.text)["data"]:
```

Comment:
The iteration variable name should be singular "day", or maybe use "date" instead
```suggestion
    for date in json.loads(get_logBook_response.text)["data"]:
```